nuntium/templates/nuntium/profiles/messages_per_instance.html

```diff
@@ -21,56 +21,76 @@
   <div class="page-header">
     <h2>{% trans "Messages" %}</h2>
   </div>
+  <div class="manager-overview">
+    {% blocktrans count message_count=writeitinstance.message_set.count %}
+    <strong>1</strong> message in your WriteIt
+    {% plural %}
+    There are <strong>{{ message_count }}</strong> messages in your WriteIt
+    {% endblocktrans %}
+  </div>
     {% autopaginate message_list %}
       {% paginate %}
       <table class="table table-striped">
         <thead>
           <tr>
-            <th>{% trans "Subject" %}</th>
-            <th>{% trans "Content" %}</th>
-            <th>{% trans "Public?" %}</th>
-            <th>{% trans "Confirmed" %}</th>
+            <th>{% trans "Message" %}</th>
+            <th class="text-center">{% trans "Public" %}</th>
+            <th class="text-center">{% trans "Confirmed" %}</th>
             {% if writeitinstance.config.moderation_needed_in_all_messages %}
-            	<th>{% trans "Moderated" %}</th>
-
+            	<th class="text-center">{% trans "Moderated" %}</th>
             {% endif %}
-            <th>{% trans "Details" %}</th>
-            <th>{% trans "Answers"%}</th>
-            <th>{% trans "Delete"%}</th>
+            <th class="text-center">{% trans "Answers"%}</th>
+            <th class="text-center">{% trans "Delete"%}</th>
           </tr>
         </thead>
         <tbody>
           {% for message in message_list %}
-          <tr>
+          <tr class="message-in-message-list">
             <td>
-              {{ message.subject }}
-              {% if message.public and message.confirmated %}
-              <a target="_blank" href="{% url 'message_detail' slug=message.slug instance_slug=writeitinstance.slug %}"><i class="fa fa-external-link"></i></a>
+              <h3><a href="{% url 'message_detail_private' subdomain=writeitinstance.slug pk=message.pk %}">{{ message.subject }}</a></h3>
+              <p class="message-in-message-list__meta">
+                {% blocktrans with created=message.created|timesince author_name=message.author_name %}
+                  Sent <strong>{{ created }}</strong> ago to <strong>{{ author_name }}</strong> from <strong>{{ author_name }}</strong>
```

Comment:
author_name in one instance here needs to be 'from' as discussed with @chrismytton